lua/gk.lua

```diff
@@ -0,0 +1,21 @@
+local ffi = require("ffi")
+local gk_config = require("gk_conf")
+local gatekeeperc = require("gatekeeperc")
+
+local M = {}
+
+-- Function that sets up the GK functional block.
+function M.setup_block()
+	local ret = 0
+
+	-- Init the GK configuration structure.
+	conf = gatekeeperc.alloc_and_return_gk_conf()
+	conf.lcore_start_id = gk_config.lcores.lcore_start_id
```

Comment:
I suggested this change in my review, with the idea that if you want multiple versions of the configuration parameters to be available for use, you can just change the `require` to drop in a different one.